### ISSUE TITLE
Suggestions should include both internals and externals

### ISSUE THREAD

**Issue by lgalke** [locdb/locdb-frend#47]
- [ ] Incorporate external suggestions in suggestion list
- [ ] Recheck that a new Resource is created, when external suggestion is chosen from list
- [ ] Drop `ngx-popover` dependency
- [ ] Think about conditionally showing external suggestions (via *show more* button)

**Comment by lgalke** [comment]
What about the following structure

# Suggestions

##  Internal

* Best match (pre-selected)
* 2nd best match
* ...
* 5th best match
* *Show More*

##  External
* Best match (pre-selected)
* 2nd best match
* ...
* 5th best match
* *Show More*

**Comment by anlausch** [comment]
+1
What do the others think?

**Comment by lgalke** [comment]
Number of displayed matches could be even user configured at some point. The external resource is in this case not a bibliographic resource yet. Should we implement the `behind-the-scenes` creation of an new target resource in the front-end or in the back-end? @anlausch

**Comment by anlausch** [comment]
Right. I remember you wanted to use the service POST /bibliographicResource for this reason (?). First selecting the external match or merging several externals according to what is the same resource, which you have already implemented (?) and then just posting?
My status is that we want to create the target resource but we do not want to add everything external that we can fetch to the database, just the ones that are actual target resources. I think the merging cannot be implemented purely in the backend because the user needs to take care of/ watch carefully everything that happens - at least a quick check of the merging result, before it is saved to the database.
If you need another service from the backend, let me know!

**Comment by anlausch** [comment]
An additional service that could maybe help you might be something like POST /addTarget. Then the back-end can take care of setting the 'cites' property add the source BR and pointing the BE to the target BR and creating the target BR. But, still the front-end would have to provide the full target BR.

**Comment by lgalke** [comment]
Alright, then we will do it in the front-end. This /addTarget service is not necessary right now.
However, there is still an issue with the `cites` property. See #51

**Comment by lgalke** [comment]
@anlausch is there a way to determine from which source the external suggestions come? We might need some kind of `provenance` field (could be even a top-level resource property).

**Comment by anlausch** [comment]
The source can be identified over the values in the identifiers array, e.g. URL_GOOGLE_SCHOLAR

```
{
    "identifiers": [
      {
        "literalValue": "http://www.sciencedirect.com/science/article/pii/S0370269398003773",
        "scheme": "URL_GOOGLE_SCHOLAR"
      }
    ],
    "title": "Gauge theory correlators from non-critical string theory",
},
```

**Comment by lgalke** [comment]
Okay that's what we did so far. Still, after some visual inspection we observed that the scheme field is filled with just `doi`. We need to check again.

**Comment by anlausch** [comment]
Can you provide a reproducable example?

**Comment by lgalke** [comment]
Nvm, I just discovered that we simply took the scheme of the wrong identifer.
```js
 {
    "identifiers": [
      {
        "literalValue": "034571179971",
        "scheme": "DOI?"
      },
      {
        "scheme": "URL_SWB",
        "literalValue": ""
      }
    ],
    "title": "Janáček - Smetana /",
    "contributors": [
      {
        "roleType": "PUBLISHER",
        "heldBy": {
          "nameString": "Hyperion Records Ltd, "
        }
      }
    ],
    "publicationYear": "2015",
    "status": "EXTERNAL"
  },
//[...]
```
Can we rely on using always the *last* Identifer's scheme?

**Comment by lgalke** [comment]
@anlausch , we did now incorporate both internal and external lists in the immediate suggestion window. However I am not 100% satisfied. The external suggestions pop-in very slowly and thus the overall look-and-feel is not really smooth. Is there a way you could speed it up (async but incomplete answers would be totally fine, or maybe just use SWB in the first place)? Else might need to think of another solution.
